install/debian-installation-guide.md

```diff
@@ -0,0 +1,45 @@
+# Install Kata Containers on Debian
+
+1. Dependencies
+
+  We need a newer version of librdbd1 to satisfy requirements: https://packages.debian.org/sid/librbd1
+
+  Add `unstable` repo to `/etc/apt/sources.list.d/unstable.list` sources list:
+  
+  ```bash
+  $ sudo sh -c "echo '# for unstable packages
+  deb http://ftp.debian.org unstable main contrib non-free
+  deb-src http://ftp.debian.org unstable main contrib non-free' > /etc/apt/sources.list.d/unstable.list"
+  ```
+  
+  Set it with a lower priority than stable, edit/create `/etc/apt/preferences.d/unstable`:
```

Comment:
Suggested rewrite:
Set the repository to a lower priority than stable, to ensures that APT will prefer stable packages over unstable ones. This can be specified in `/etc/apt/preferences.d/unstable`: